src/toggle/toggle.styles.tsx

```diff
@@ -0,0 +1,187 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { Text, TextStyle, TextStyleHelper, TextWeight } from "../text";
+import { ToggleStyleType } from "./types";
+
+// =============================================================================
+// STYLE INTERFACES, transient props are denoted with $
+// See more https://styled-components.com/docs/api#transient-props
+// =============================================================================
+interface StyleProps {
+    $selected?: boolean;
+    $disabled?: boolean;
+    $indicator?: boolean;
+}
+
+interface ContainerStyleProps extends StyleProps {
+    $styleType?: ToggleStyleType;
+    $error?: boolean;
+    $hasSubLabel?: boolean;
+}
+
+interface LabelStyleProps extends StyleProps {}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const Container = styled.div<ContainerStyleProps>`
+    position: relative;
+    display: inline-flex;
+    min-width: 10.375rem;
+    border-radius: 4px;
+    border-width: 1px;
+    border-style: solid;
+    cursor: ${(props) => (props.$disabled ? "not-allowed" : "pointer")};
+    padding: 0.6875rem 1rem;
+
+    // Content positioning style
+    ${(props) => {
+        if (!props.$indicator) {
+            return css`
+                justify-content: center;
+            `;
+        }
+    }}
+
+    // Background, Hover and Border style
+    ${(props) => {
+        switch (props.$styleType) {
+            case "no-border": {
+                if (props.$error) {
+                    return css`
+                        border-color: ${Color.Validation.Red.Icon};
```

Comment:
just want to double check that the toggle bg remains transparent here? and also has no bg on hover?

Reply:
It should have. Will add back. Missed out.